R/convenience-functions.R

```diff
@@ -0,0 +1,48 @@
+#' @title Add transformations
+#'
+#' @description Add transformations of the forecasts and observations for
+#' later scoring. For more information on why this might be desirable, check
+#' out the linked reference.
+#'
+#' @inheritParams score
+#' @param fun A function used to transform both true values and predictions
+#' @param label A string for the newly created 'scale' column to denote the
```

Comment:
If you are renaming these columns how do you expect this to work with score? I think you should be showing this workflow here as an example? 

I assumed the logical thing to do was transform the existing columns so you can score them and keep a copy as natural scale?